tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -69,6 +72,71 @@ Test Metric "Active_Users" On OpenShift Monitoring On Cluster Monitoring Prometh
     Should Be Equal    ${value}    2    msg=Active Users are not equal to length of list or N users
     [Teardown]    CleanUp JupyterHub For N users    list_of_usernames=${list_of_usernames}
 
+Test Metric "Active Users" On Telemeter
+    [Documentation]    Verifies the openshift metrics and telemeter shows
+    ...                the same rhods active users
+    [Tags]    ODS-1054
+    ...       Tier1
+     @{list_of_usernames} =    Create List    ${TEST_USER_3.USERNAME}    ${TEST_USER_4.USERNAME}
+    Log In N Users To JupyterLab And Launch A Notebook For Each Of Them
+    ...    list_of_usernames=${list_of_usernames}
+    ${value} =    Run OpenShift Metrics Query    query=${METRIC_RHODS_ACTIVE_USERS}
+    ${cluster_id} =    Get Cluster ID
+    ${time} =    Get Start Time And End Time    interval=15m
+    ${steps} =    Set Variable    15
+    ${query} =    Set Variable    cluster:usage:consumption:rhods:active_users{_id=${cluster_id}}
+    ${url}=    Set Variable    ${telmeter_url}query_range?query=${query}&start=${time[0]}&end=${time[1]}&step=${steps}
+    Open Browser     ${url}    ${BROWSER.NAME}
+    Click Button    //button[@type="submit"]
+    Click Element    //a[@title="Log in with SSO"]
+    @{data} =    Get WebElements    //pre
+    &{data} =    Evaluate    dict(${data[0].text})
+    Close Browser
+    ${data} =    Set Variable    ${data["data"]["result"][0]["values"][-1][1]}
```

Comment:
Can this be made into a Keyword?
e.g. `Run Query And Get Result From Telemeter`